Tao_Current.txt

```diff
@@ -444,32 +507,36 @@ You can always tell a person who doesn't understand the IETF (or is intentionall
 
 When you submit an Internet-Draft, you give some publication rights to the IETF. This is so that your Internet-Draft is freely available to everyone who wants to read and comment on it. The rights you do and don't give to the IETF are described in [BCP78], "IETF Rights in Contributions".
 
-There is a very useful checking tool at http://tools.ietf.org/tools/idnits. Using this tool before you turn in an Internet-Draft will help prevent the draft from being rejected due to errors in form and formatting.
+There is a very useful checking tool at [https://tools.ietf.org/tools/idnits]. Using this tool before you turn in an Internet-Draft will help prevent the draft from being rejected due to errors in form and formatting.
 
-An I-D should have approximately the same format as an RFC. Contrary to many people's beliefs, an I-D does not need to look exactly like an RFC, but if you can use the same formatting procedures used by the RFC Editor when you create your I-Ds, it will simplify the RFC Editor's work when your draft is published as an RFC. [RFC2223], "Instructions to RFC Authors", describes the submission format. There is also a tool called "xml2rfc", available from http://xml.resource.org, that takes XML-formatted text and turns it into a valid Internet-Draft.
+An I-D should have approximately the same format as an RFC. Contrary to many people's beliefs, an I-D does not need to look exactly like an RFC, but if you can use the same formatting procedures used by the RFC Editor when you create your I-Ds, it will simplify the RFC Editor's work when your draft is published as an RFC. [RFC2223], "Instructions to RFC Authors", describes the submission format. There is also a tool called "xml2rfc", available from [https://xml2rfc.tools.ietf.org], that takes XML-formatted text and turns it into a valid Internet-Draft.
```

Comment:
Given how many people use markdown now, you might want to cite that.

Reply:
An I-D should have approximately the same format as an RFC. Contrary to many people's beliefs, an I-D does not need to look exactly like an RFC, but if you can use the same formatting procedures used by the RFC Editor when you create your I-Ds, it will simplify the RFC Editor's work when your draft is published as an RFC. [RFC2223], "Instructions to RFC Authors", describes the submission format. There is also a tool called "xml2rfc", available from [https://xml2rfc.tools.ietf.org], that takes XML-formatted text and turns it into a valid Internet-Draft, you can also use a tool called kramdown, available from [https://github.com/cabo/kramdown-rfc2629], that takes markdown-formatted text and turns it into a valid Internet-Draft.